netty/src/test/java/io/grpc/netty/NettyStreamTestBase.java

```diff
@@ -199,7 +200,7 @@ public void shouldNotBeReadyForDataAfterWritingLargeMessage() throws IOException
   }
 
   protected byte[] smallMessage() {
-    return MESSAGE.getBytes();
+    return MESSAGE.getBytes(US_ASCII);
```

Comment:
Like most other refs, a 

private static final US_ASCII = Charset.forName("US-ASCII);


is probably good enough.  Like `@Beta`, we probably shouldn't depend on `@Deprecated` stuff.